apps/server/src/config/manifest.ts

```diff
@@ -0,0 +1,3 @@
+import pkg from '../../package.json' assert { type: 'json' };
+
+export const { name, version } = pkg;
```

Comment:
👍 nice encapsulation. all those other files shouldn't have access to the rest of the manifest.
thanks for making this change